material/30days-curriculum/day01_開発環境を整えて、初めてのアプリを動かそう.md

```
@@ -234,14 +270,14 @@ pwd
 ls
 ```
 
-`ls` は、いまいるフォルダの中身を並べて見せるだけのコマンドです。ファイルを作ったり消したりしないので、迷子になったと感じたときは何度でも実行できます。一覧に `scripts` と `material` が出てこないときは、展開先の1つ外側か1つ内側にいます。表示の中に `task-app` というフォルダ名が見えていれば `cd task-app` で中に入り、見覚えのない一覧が出たときは `cd ~/workspace/task-app` と打てば戻ってこられます。
+`ls` は、いまいるフォルダの中身を並べて見せるだけのコマンドです。ファイルを作ったり消したりしないので、迷子になったと感じたときは何度でも実行できます。一覧に `README.md` と `scripts` が出てこないときは、展開先の1つ外側か1つ内側にいます。表示の中に `task-app` というフォルダ名が見えていれば `cd task-app` で中に入り、見覚えのない一覧が出たときは `cd ~/workspace/task-app` と打てば戻ってこられます。
 
-### 期待する結果
+#### 期待する結果
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)